lazyflow/utility/helpers.py

```diff
@@ -127,3 +129,37 @@ def bigintprod(nums: Iterable[numbers.Integral]) -> int:
     array shapes seen in practice on some operating systems (windows: long -> 32 bit).
     """
     return reduce(mul, map(int, nums))
+
+
+def get_ram_per_element(dtype: Union[object, numpy.dtype]) -> int:
```

Comment:
By the way, the type signature is somewhat incorrect: `numpy.dtype()` accepts a lot of stuff, and also `numpy.dtype` value is an `object` too.

Reply:
yeah  right. It should be... `Type[object]` I suppose

Reply:
Strictly speaking. it should be [`numpy.typing.DtypeLike`](https://numpy.org/doc/stable/reference/typing.html#numpy.typing.DTypeLike).

Reply:
Awesome, thanks for the heads-up there! Looks like it was only introduced in numpy 1.20 - which we currently don't support because of vigra. Let's keep this in mind until we can support numpy 1.20. Will leave it at that for the time being.